pyluos/device.py

```diff
@@ -370,21 +370,21 @@ def __init__(self, host,
 
     @property
     def set(self):
-        self._send({'init': {}})
+        self._send('init')
```

Comment:
those modifications could be fixuped into the commit related to this class creation.